### PR TITLE
Feature/view posts

### PR DESCRIPTION
**Pull request by therearedoors**
- some confusion around whether to push dev environment files like .env, have now removed this.
Relevant change here is the controller for getAllPosts route.

### CODE REVIEW

**Review by alsyoufomar — DISMISSED** [review]

File: .env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+PORT=4000
```
Comment (Auenc) [inline .env:1]: You don't want to commit your `.env` file. Can you remove this please?

File: src/controllers/post.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { sendDataResponse } from '../utils/responses.js'
 
 export const create = async (req, res) => {
   const { content } = req.body
-
+  console.log(req.user)
```
Comment (Auenc) [inline src/controllers/post.js:7]: if this line is not needed anymore, then it would be a good idea to remove it

File: src/controllers/user.js [hunk 3]
```diff
@@ -12,6 +12,7 @@ export const create = async (req, res) => {
     }
 
     const createdUser = await userToCreate.save()
+    console.log(createdUser)
```
Comment (Auenc) [inline src/controllers/user.js:15]: Same as above, if not needed, remove this line :)

**Review by Auenc — CHANGES_REQUESTED** [review]
Biggest changes needed here are:
- remove `.env` 
- re-add `.env.example`

**Review by phillipdunn — APPROVED** [review]
LGTM

**Review by Auenc — APPROVED** [review]
LGTM

**Review by phillipdunn — APPROVED** [review]

**Pull request merged**